tests/acceptance/EnableAndDisableSignupConfirmationCest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use Codeception\Util\Locator;
+use Facebook\WebDriver\WebDriverKeys;
+use MailPoet\Test\DataFactories\Form;
+
+require_once __DIR__ . '/../DataFactories/Form.php';
+
+class EnableAndDisableSignupConfirmationCest {
+
+  private function setSignupConfirmationSetting(\AcceptanceTester $I, $enabled) {
+    $choice_selector = $enabled ? 
+      '[data-automation-id="enable_signup_confirmation"]' :
+      '[data-automation-id="disable_signup_confirmation"]';
+    $I->amOnMailPoetPage('Settings');
+    $I->click('[data-automation-id="signup_settings_tab"]');
+    $I->waitForText('Enable sign-up confirmation');
+    $I->click($choice_selector);
+    $I->acceptPopup();
+    $I->click('[data-automation-id="settings-submit-button"]');
+  }
+
+  private function addFormWidget(\AcceptanceTester $I) {
+    $form_factory = new Form();
+    $form = $form_factory->withName('Confirmation Form')->create();
+    $I->cli('widget reset sidebar-1 --allow-root');
+    $I->cli('widget add mailpoet_form sidebar-1 2 --form=' . $form->id . ' --title="Subscribe to Our Newsletter" --allow-root');
+  }
+
+  private function subscribeUsingWidgetForm(\AcceptanceTester $I) {
+    $I->amOnUrl('http://wordpress');
```

Comment:
I don't think it is a good idea to spread this out. We should at least use a constant for all the test in case it is changed. Would this work? `$this->amOnPage('/');`

Reply:
`$this->amOnPage('/');` will not work because I am on the `mailhog` domain. I can:

- add a constant to `AcceptanceTester` so that we use it on all tests.
- or override the `amOnPage` method to prepend the domain to the URL.

what do you think?

Reply:
I think the constant should be fine.